components/ReviewPanel.tsx

```
@@ -0,0 +1,247 @@
+    return targets.sort((a, b) => a.localeCompare(b));
+  }, [selectedSubs, postToProfile, userName]);
+
+  React.useEffect(() => {
+    if (!mediaFiles || mediaFiles.length === 0) {
+      setMediaPreviews([]);
+      return;
+    }
+    const previews = mediaFiles.map((file) => ({ url: URL.createObjectURL(file), type: file.type }));
+    setMediaPreviews(previews);
+    return () => {
+      previews.forEach((preview) => URL.revokeObjectURL(preview.url));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)